src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -1412,7 +1412,7 @@ private uint getClassSize(CORINFO_CLASS_STRUCT_* cls)
         private uint getClassAlignmentRequirement(CORINFO_CLASS_STRUCT_* cls, bool fDoubleAlignHint)
         {
             DefType type = (DefType)HandleToObject(cls);
-            return (uint)type.InstanceByteAlignment.AsInt;
```

Comment:
I have reviewed all places that call InstanceByteCount or InstanceByteAlignment and found a few more places that were calling it incorrectly in addition to the one place in interop.